lambda-resume-builder/src/Components/Dashboard.js

```diff
@@ -167,22 +174,22 @@ class Dashboard extends React.Component {
         </Drawer>
         <main className={classes.content}>
           <div className={classes.appBarSpacer} />
-          <Typography variant="h4" gutterBottom component="h2">
-            Placeholder
-          </Typography>
-          <Typography component="div" className={classes.chartContainer}>
-
-          </Typography>
-          <Typography variant="h4" gutterBottom component="h2">
-            Placeholder
-          </Typography>
+          <Router>
+            <Route exact path="/dashboard/assignments" render = {props => {
+            return (<Assignments {...props}/>)
```

Comment:
### I really prefer having a route to the checklist here first. It feels like it was much better designed and implemented than the assignments or calendar components.

### Also, I really think that all of this could use more OAuth @AntonioUniverse if you could get on that I'd really appreciate it🍆